File: 283-MoveZeroes/283-MoveZeroes.c
// Last updated: 1/6/2026, 5:33:58 PM
void moveZeroes(int* nums, int numsSize) {
    int left = 0;

    for (int i = 0; i < numsSize; i++) {
        if (nums[i] != 0) {
            int temp = nums[left];
            nums[left] = nums[i];
            nums[i] = temp;
            left += 1;
        }
    }
}

File: 283-MoveZeroes/283-MoveZeroes.py
# Last updated: 1/6/2026, 5:34:00 PM
class Solution(object):
    def moveZeroes(self, nums):
        left = 0

        for i in range(len(nums)):
            if nums[i] != 0:
                temp = nums[left]
                nums[left] = nums[i]
                nums[i] = temp
                left += 1